### PR TITLE
Beer-Song: Compliance with JSON Schema

### PR DESCRIPTION
**Pull request by kotp**
Adjusts #661 for #625

### CODE REVIEW

**Comment by kotp** [comment]
@kytrinyx If this looks good to you, could you merge it into your PR #661

File: exercises/beer-song/canonical-data.json [hunk 1]
```diff
@@ -1,55 +1,77 @@
...
-      }
-    ]
-  }
+  "exercise": "beer-run",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "verse",
+      "cases": [
```
Comment (kytrinyx) [inline exercises/beer-song/canonical-data.json]: Is the "cases" => "cases" nesting done on purpose?

Comment (kytrinyx) [inline exercises/beer-song/canonical-data.json]: Oh, wait. Yeah, I get it.

**Comment by kytrinyx** [comment]
Thanks, @kotp! I remember seeing the schema discussion go by, but haven't paid close attention. This is very helpful.

**Pull request merged**